Okey, I f*ked something up when commiting stuff, now it seems to have problems with loading config - I didn't even have to touch that for .19.3 migration. 

Edit: I did rebase my changes onto the  `fabric-1.19.1` branch before commiting, was working on top of `3.0.0-8` tag previously 🤔

```gradle
Starting net.fabricmc.loader.impl.game.minecraft.BundlerClassPathCapture
[06:49:42] [main/INFO]: Loading Minecraft 1.19.3 with Fabric Loader 0.14.11
[06:49:42] [main/INFO]: Loading 56 mods:
        - at_favre_lib_bcrypt 0.9.0 via easyauth
        - at_favre_lib_bytes 1.5.0 via easyauth
        - de_mkammerer_argon2-jvm 2.11 via easyauth
        - de_mkammerer_argon2-jvm-nolibs 2.11 via easyauth
        - easyauth 3.0.0-11
        - fabric-api 0.68.1+1.19.3
        - fabric-api-base 0.4.19+8d1895cf72 via fabric-api
        - fabric-api-lookup-api-v1 1.6.18+49abcf7e72 via fabric-api
        - fabric-biome-api-v1 12.0.0+689f5e7172 via fabric-api
        - fabric-block-api-v1 1.0.3+12bfe4ea72 via fabric-api
        - fabric-command-api-v1 1.2.19+f71b366f72 via fabric-api
        - fabric-command-api-v2 2.1.15+49abcf7e72 via fabric-api
        - fabric-commands-v0 0.2.36+df3654b372 via fabric-api
        - fabric-containers-v0 0.1.44+df3654b372 via fabric-api
        - fabric-content-registries-v0 3.4.11+49abcf7e72 via fabric-api
        - fabric-convention-tags-v1 1.1.9+2894d6df72 via fabric-api
        - fabric-crash-report-info-v1 0.2.12+aeb40ebe72 via fabric-api
        - fabric-data-generation-api-v1 11.0.1+65e415cb72 via fabric-api
        - fabric-dimensions-v1 2.1.39+48349a3f72 via fabric-api
        - fabric-entity-events-v1 1.5.5+b83334a072 via fabric-api
        - fabric-events-interaction-v0 0.4.37+422b77fb72 via fabric-api
        - fabric-events-lifecycle-v0 0.2.38+df3654b372 via fabric-api
        - fabric-game-rule-api-v1 1.0.28+2894d6df72 via fabric-api
        - fabric-item-api-v1 2.1.4+fa25da9972 via fabric-api
        - fabric-item-group-api-v1 2.1.6+b7096da872 via fabric-api
        - fabric-lifecycle-events-v1 2.2.8+2894d6df72 via fabric-api
        - fabric-loot-api-v2 1.1.17+75e9821172 via fabric-api
        - fabric-loot-tables-v1 1.1.21+9e7660c672 via fabric-api
        - fabric-message-api-v1 5.0.12+6ede1da972 via fabric-api
        - fabric-mining-level-api-v1 2.1.28+49abcf7e72 via fabric-api
        - fabric-networking-api-v1 1.2.14+d9ac3c3972 via fabric-api
        - fabric-networking-v0 0.3.31+df3654b372 via fabric-api
        - fabric-object-builder-api-v1 5.1.6+49abcf7e72 via fabric-api
        - fabric-particles-v1 1.0.18+fbde993d72 via fabric-api
        - fabric-permissions-api-v0 0.1-SNAPSHOT via easyauth
        - fabric-registry-sync-v0 2.0.1+2608564672 via fabric-api
        - fabric-rendering-data-attachment-v1 0.3.23+48349a3f72 via fabric-api
        - fabric-rendering-fluids-v1 3.0.16+3067f46372 via fabric-api
        - fabric-resource-conditions-api-v1 2.2.0+280be3ab72 via fabric-api
        - fabric-resource-loader-v0 0.10.2+f6c919d672 via fabric-api
        - fabric-screen-handler-api-v1 1.3.10+689f5e7172 via fabric-api
        - fabric-transfer-api-v1 2.1.11+28ac448f72 via fabric-api
        - fabric-transitive-access-wideners-v1 2.1.0+0efcd39b72 via fabric-api
        - fabricloader 0.14.11
        - java 17
        - minecraft 1.19.3
        - mysql_mysql-connector-java 8.0.29 via easyauth
        - net_java_dev_jna_jna 5.12.1 via easyauth
        - org_iq80_leveldb_leveldb 0.12 via easyauth
        - org_iq80_leveldb_leveldb-api 0.12 via easyauth
        - org_mongodb_bson 4.7.0 via easyauth
        - org_mongodb_mongodb-driver-core 4.7.0 via easyauth
        - org_mongodb_mongodb-driver-sync 4.7.0 via easyauth
        - packet_tweaker 0.3.0+1.18.2 via server_translations_api
        - placeholder-api 2.0.0-beta.7+1.19 via easyauth
        - server_translations_api 1.4.16+1.19 via easyauth
[06:49:42] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/P:/PersonalFolders/Desktop/fabric/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[06:49:42] [main/INFO]: Compatibility level set to JAVA_16
[06:49:42] [main/INFO]: Compatibility level set to JAVA_17
[06:49:46] [main/INFO]: Building unoptimized datafixer
[06:49:47] [main/INFO]: [EasyAuth] EasyAuth mod by samo_lego, NikitaCartes.
[06:49:47] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'easyauth'!
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.14.11.jar:?]
        at net.minecraft.server.Main.main(Main.java:102) ~[server-intermediary.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.14.11.jar:?]
        at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:69) ~[fabric-server-mc.1.19.3-loader.0.14.11-launcher.0.11.1.jar:0.11.1]
Caused by: java.lang.NullPointerException: Cannot read field "experimental" because "xyz.nikitacartes.easyauth.EasyAuth.config" is null
        at xyz.nikitacartes.easyauth.storage.database.LevelDB.<init>(LevelDB.java:30) ~[easyauth-mc1.19.3-3.0.0-11.jar:?]
        at xyz.nikitacartes.easyauth.storage.AuthConfig.load(AuthConfig.java:102) ~[easyauth-mc1.19.3-3.0.0-11.jar:?]
        at xyz.nikitacartes.easyauth.EasyAuth.init(EasyAuth.java:80) ~[easyauth-mc1.19.3-3.0.0-11.jar:?]
        at xyz.nikitacartes.easyauth.EasyAuth.onInitialize(EasyAuth.java:107) ~[easyauth-mc1.19.3-3.0.0-11.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.11.jar:?]
        ... 8 more
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,4 @@
-### 3.0.0 Minecraft 1.17, 1.17.1, 1.18[.1], 1.18.2, 1.19, 1.19.1
+### 3.0.0 Minecraft 1.17, 1.17.1, 1.18[.1], 1.18.2, 1.19, 1.19.1-1.19.2, 1.19.3
 
 1) Fix MySQL support
 2) Add placeholder api support, thanks to @Nikijaz
@@ -12,6 +12,8 @@
 10) Improve kicking if number of login attempts exceeded
 11) Save EasyAuth's database on `save-all` command
 12) Reconnect to DB with `auth reload`
+13) Fix `hideUnauthenticatedPLayersFromPlayerList`, thanks to @Wereii
+14) Update Simplified Chinese Localization, thanks to @GodGun968
 
 ### 2.2.2, Minecraft 1.17, 1.17.1, 1.18[.1], 1.18.2
 

--- a/README.md
+++ b/README.md
@@ -18,11 +18,13 @@ See [wiki](https://github.com/NikitaCartes/EasyAuth/wiki) for more information.
 * Thanks to @were491 for improvement in login system
 * Thanks to @Nikijaz for implementing MySQL database support
 * Thanks to @@dmunozv04 for Floodgate support
+* Thanks to @dmunozv04 for Floodgate support
+* Thanks to @Wereii for 1.19.3 support
 
 Thank you so much for those who have already helped with translation of the mod!
 
 * @alphatoasterous (Russian)
-* @Neubulae (Chinese)
+* @Neubulae, @GodGun968 (Chinese)
 * @Zailer43 (Spanish)
 * @Kljunas2 (Slovene)
 * @Bendimester23 (Hungarian)

